recipes/gromacs_mpi/README.md

```diff
@@ -0,0 +1,24 @@
+This recipe is for the MPI enabled version of GROMACS.  For the non MPI
+enebaled version, use the GROMACS package.
+
+GROMACS versions are released in series, so we maintain one
+recipe folder for each major version.
+
+Be careful with the CI build time if updating multiple 
+recipes, a single build takes about 1h12.
```

Comment:
Can you clarify "single build?" There's 4 SIMD configurations by 2 MPI configurations built in this job.

Reply:
Not able to, that has been in the README since before I started adding new versions. I can remove it.